### PR TITLE
Formal docs fixes

### PR DESCRIPTION
**Pull request by andy-maier**
See commit message.

### CODE REVIEW

File: docs/pywbemcli/features.rst [hunk 1]
```diff
@@ -95,21 +95,24 @@ DeleteQualifier                    Not Implemented
...
-are to be applied to CIM instances and methods to be sent to the WBEM server.
+The :ref:`instance create command`, :ref:`instance modify command`,
+:ref:`class invokemethod command`, and :ref:`instance invokemethod command`
+define the values of properties and parameters that are to be applied to CIM
+instances and methods to be sent to the WBEM server.
 
-For a single property or parameter this is the ``--property``/``-p`` or
-``--parameter``/``-p`` option with the name and value in the form:
+For a single property or parameter these are the ``--property``/``-p`` or
+``--parameter``/``-p`` command options with its name and value in the form:
```
Comment (KSchopmeyer) [inline docs/pywbemcli/features.rst:104]: Change "with its name and value" to "with name and value"

File: docs/pywbemcli/generaloptions.rst [hunk 2]
```diff
@@ -881,87 +881,57 @@ Output formats for groups and commands
...
-+----------+---------------+----------+----------------+--------------------------------------------+
+default format for the :ref:`--output-format general option`.
+
+The following shows the default format for each command and the alternate
+formats where the values mean:
+
+objects: ``xml``, ``repr``, or ``txt``
+
+table: ``table``, ``plain``, ``simple``, ``grid``, ``psql``, ``rst``, or ``html``
+
```
Comment (KSchopmeyer) [inline docs/pywbemcli/generaloptions.rst:892]: I forgot to add 'text' to this

File: docs/pywbemcli/mock_support.rst [hunk 3]
```diff
@@ -10,12 +10,12 @@ Mock WBEM server support
 Mock support overview
 ---------------------
 
-Pywbemcli has support for mocking a WBEM server with the general option
-:ref:`--mock-server general option` (``--mock-server/-m``) . This allows
-executing pywbemcli against a mock WBEM server that is automatically created in
-pywbemcli, rather than a real WBEM server.
+Pywbemcli has support for mocking a WBEM server with the
```
Comment (KSchopmeyer) [inline docs/pywbemcli/mock_support.rst:13]: Change has support for to supports

File: docs/pywbemcli/generaloptions.rst [hunk 4]
```diff
@@ -881,87 +881,57 @@ Output formats for groups and commands
...
+========== ============= ======== ============== ============================================
```
Comment (KSchopmeyer) [inline docs/pywbemcli/generaloptions.rst:893]: This reformat resolves several issues.   Note that there is a table directive that puts a title on the table. I assume that it also allows creating references to the table but have not quite sorted that out.  Using that
directive for first time in pywbem doc update today. Turns out the old format really does not work well with the table directive so just changed the one in pywbem

File: docs/pywbemcli/features.rst (hunk 1 above)
Comment (andy-maier) [reply docs/pywbemcli/features.rst:104]: DONE

File: docs/pywbemcli/generaloptions.rst (hunk 2 above)
Comment (andy-maier) [reply docs/pywbemcli/generaloptions.rst:892]: DONE

File: docs/pywbemcli/generaloptions.rst [hunk 5]
```diff
@@ -1169,8 +1139,8 @@ The output of CIM objects allows multiple formats as follows:
   This should be considered the output of last resort as it simply uses
   the ``__str__()`` method of the Python class for each CIM object to output.
 
-  Thus, for example the a ``class enumerate`` of a model with only a single
-  class is of the form:
+  Thus, for example a ``class enumerate`` command of a model with only a single
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcli/generaloptions.rst:1142]: DONE

File: docs/pywbemcli/mock_support.rst (hunk 3 above)
Comment (andy-maier) [reply docs/pywbemcli/mock_support.rst:13]: DONE

**Pull request merged**